data/simulations/FAH-Mpro-monomer-no-water.yml

```diff
@@ -0,0 +1,60 @@
+type: md
+title: SARS-CoV-2 main protease (apo, monomer) Folding@home simulations
+description: >
+  Folding@home projects 11743 and 11749 are made available here without solvent, but including salt.
```

Comment:
Lead with info on what the data is, rather than project numbers.

Reply:
Lead with something like:

This is a dataset containing N simulations up to 1 microsecond in length of SARS-CoV-2 main viral protease (Mpro, or 3CL) in its apo, monomeric form.

Simulations were constructed from ...